coordinator/core/marbleapi.go

```diff
@@ -177,26 +212,42 @@ func (c *Core) generateCertFromCSR(csrReq []byte, pubk ecdsa.PublicKey, marbleTy
 }
 
 // customizeParameters replaces the placeholders in the manifest's parameters with the actual values
-func customizeParameters(params *rpc.Parameters, rootCA []byte, marbleCert []byte, marbleKey []byte, sealKey string) *rpc.Parameters {
+func customizeParameters(params *rpc.Parameters, marbleSecretsWrapped marbleSecretsWrapper) (*rpc.Parameters, error) {
```

Comment:
add a unit test for this function

Reply:
`TestActivate` does already test the reserved secrets and will fail if e.g. the generated rootCA PEM is invalid and cannot be decoded correctly. 

Considering support for other, non-reserved secrets is coming in an upcoming PR, I'd suggest we create a separate unit test then which will then test the general implementation of the templates. But for the specific case of the "reserved" secrets in this PR, it should already be covered.